src/interfaces/Grant.ts

```diff
@@ -23,6 +23,7 @@ export interface VestingInfo {
   vesting_token_contract_balance: number
   vesting_total_amount: number
   vesting_status: VestingStatus
+  duration: number
```

Comment:
I'm thinking this name should explicit the units of the duration somehow. 

How about `months_duration` or `duration_in_months`?